com.playeveryware.eos/Runtime/Core/Config/JsonConverter/ListOfStringsToEnumConverters.cs

```diff
@@ -36,6 +36,7 @@ namespace PlayEveryWare.EpicOnlineServices
     using Newtonsoft.Json.Linq;
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics;
```

Comment:
This is an unused using statement - typically it's introduced when Visual Studio assumes by `Debug` you mean `System.Diagnostics.Debug`.